src/server/schemas/friends.schema.ts

```diff
@@ -0,0 +1,15 @@
+
+import  mongoose from 'mongoose';
+import { Friends} from '../../shared/models/friends.model.js';
+
+
+
+const {Schema, model} = mongoose
+
+const friendsSchema = new Schema<Friends>({
+user: {type: mongoose.Types.ObjectId},
+text: {type: String, required: true},
+ 
+})
+
+export const FriendsModel = model<Friends>('Friends',friendsSchema)
```

Comment:
would love it if you could give the structure of this new collection some thought .. i dont know if you are right or wrong, but i cant quite grasp what the benefit or function this collection adds, or how it will work.

Reply:
I just changed the shape of friends model and schema. Hopefully this one is a bit more clear.